### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by linsui** [huanli233/BiliClient#2]
Can I package this app for F-Droid?

**Comment by huanli233** [comment]
Of course, you can.

**Comment by linsui** [comment]
I'll have a try. :)

**Comment by linsui** [comment]
How can I build those ijk libs?

**Comment by huanli233** [comment]
> How can I build those ijk libs?

I don't know what problem you encountered during construction. But you may refer to the build commands in the [action_build](https://github.com/huanli233/action_build_biliclient) repository. And if you only need to obtain the package generated by the build, you can go to the website

（你是国人吗（（

**Comment by linsui** [comment]
> I don't know what problem you encountered during construction. But you may refer to the build commands in the [action_build](https://github.com/huanli233/action_build_biliclient) repository. And if you only need to obtain the package generated by the build, you can go to the website

F-Droid doesn't allow binaries and I need to rebuild these files in https://github.com/huanli233/BiliClient/tree/develop/app/libs/armeabi-v7a.

> （你是国人吗（（

是 :joy: 外国人也用 Bilibili 吗（

**Comment by huanli233** [comment]
> > I don't know what problem you encountered during construction. But you may refer to the build commands in the [action_build](https://github.com/huanli233/action_build_biliclient) repository. And if you only need to obtain the package generated by the build, you can go to the website
> 
> F-Droid doesn't allow binaries and I need to rebuild these files in https://github.com/huanli233/BiliClient/tree/develop/app/libs/armeabi-v7a.
> 
> > （你是国人吗（（
> 
> 是 :joy: 外国人也用 Bilibili 吗（

Perhaps you can check the repository of ijkplayer

**Comment by linsui** [comment]
Yep, but since it's your "own" ijk lib, did you apply any patches? Which version/commit should I use?

**Comment by huanli233** [comment]
Sorry for the late reply.
According to the information I understand, the ijkplayer used in this project has not been modified
For the build of ijkplayer, please refer to the action-build repository

I am no longer involved in the project development, the Github repository is only a mirror, so this issue will be closed.
If you need more information, please go to the Gitee repository or contact the developer through QQ or other channels.
